database/index.js

```diff
@@ -0,0 +1,54 @@
+// const helper = require('./helpers/dataGenerator.js')
+const mongoose = require('mongoose');
+// mongoose.connect('mongodb://localhost/products');
+mongoose.connect('mongodb://mattminwoolee:password123@ds147942.mlab.com:47942/front-end-capstone');
```

Comment:
Use dotenv and process.env.VAR to protect this sensitive and environmental vars here!